Source/Core/VideoCommon/BoundingBox.h

```diff
@@ -4,40 +4,44 @@
 
 #pragma once
 
+#include <array>
+#include <vector>
+
 #include "Common/CommonTypes.h"
 
 class PointerWrap;
 
-// Bounding Box manager
-namespace BoundingBox
-{
-// Indicates a coordinate of the bounding box.
-enum class Coordinate
+using BBoxType = s32;
+static constexpr u32 NUM_BBOX_VALUES = 4;
```

Comment:
```suggestion
constexpr u32 NUM_BBOX_VALUES = 4;
```

given constexpr internally links by default for values at file/namespace scope.